napari/utils/colormaps/colormap.py

```diff
@@ -68,6 +74,9 @@ class Colormap(EventedModel):
     _display_name: str | None = PrivateAttr(None)
     interpolation: ColormapInterpolationMode = ColormapInterpolationMode.LINEAR
     controls: Array = Field(default_factory=lambda: cast(Array, []))
+    nan_color: ColorValue = ColorValue('transparent')
+    high_color: ColorValue | None = None
+    low_color: ColorValue | None = None
```

Comment:
It looks like we could remove `__iter__` method from Colormap and stop breaking `pydatntic.BaseModel` API. (lines 138-139)

Reply:
good point!